modules/mri_violations/php/NDB_Menu_Filter_Form_mri_violations.class.inc

```diff
@@ -438,6 +370,7 @@ class NDB_Menu_Filter_Form_mri_violations extends NDB_Menu_Filter_Form
         return array_merge(
             $deps,
             array(
+                $baseurl . "/mri_violations/js/Not_Resolved_columnFormatter.js",
                 $baseurl . "/mri_violations/js/mri_violations_helper.js",
```

Comment:
This script seems to be also trying to load a react table in an old way. It probably should be removed